website/docs/d/privatelink_endpoint_service.html.markdown

```diff
@@ -61,15 +61,19 @@ In addition to all arguments above, the following attributes are exported:
 * `private_endpoint_resource_id` - Unique identifier of the private endpoint.
 * `delete_requested` - Indicates if Atlas received a request to remove the interface endpoint from the private endpoint connection.
 * `error_message` - Error message pertaining to the interface endpoint. Returns null if there are no errors.
-* `connection_status` - Status of the interface endpoint.
+* `aws_connection_status` - Status of the interface endpoint for AWS.
   Returns one of the following values:
-    * `NONE` - Atlas created the network load balancer and VPC endpoint service, but AWS hasn’t yet created the VPC endpoint.
-    * `PENDING_ACCEPTANCE` - AWS has received the connection request from your VPC endpoint to the Atlas VPC endpoint service.
-    * `PENDING` - AWS is establishing the connection between your VPC endpoint and the Atlas VPC endpoint service.
-    * `AVAILABLE` - Atlas VPC resources are connected to the VPC endpoint in your VPC. You can connect to Atlas clusters in this region using AWS PrivateLink.
-    * `REJECTED` - AWS failed to establish a connection between Atlas VPC resources to the VPC endpoint in your VPC.
-    * `INITIATING` - Atlas has not yet accepted the connection to your private endpoint.
-    * `FAILED` - Atlas failed to accept the connection your private endpoint.
-    * `DELETING` - Atlas is removing the interface endpoint from the private endpoint connection.
+  * `NONE` - Atlas created the network load balancer and VPC endpoint service, but AWS hasn’t yet created the VPC endpoint.
+  * `PENDING_ACCEPTANCE` - AWS has received the connection request from your VPC endpoint to the Atlas VPC endpoint service.
+  * `PENDING` - AWS is establishing the connection between your VPC endpoint and the Atlas VPC endpoint service.
+  * `AVAILABLE` - Atlas VPC resources are connected to the VPC endpoint in your VPC. You can connect to Atlas clusters in this region using AWS PrivateLink.
+  * `REJECTED` - AWS failed to establish a connection between Atlas VPC resources to the VPC endpoint in your VPC.
+  * `DELETING` - Atlas is removing the interface endpoint from the private endpoint connection.
```

Comment:
I noticed this above in the checks as well - did we double check these against what the docs has?  https://docs.atlas.mongodb.com/reference/api/private-endpoints-service-create-one for AWS lists:

INITIATING | Atlas is creating the network load balancer and VPC  endpoint service.
WAITING_FOR_USER | The Atlas network load balancer and VPC endpoint service are created and ready to receive connection requests. When you receive this status, create an interface endpoint to continue configuring the AWS PrivateLink connection.
FAILED | A system failure has occurred.
DELETING | The AWS PrivateLink connection is being deleted.

It looks like this and the code did not get updated from the deprecated resources (https://docs.atlas.mongodb.com/reference/api/private-endpoint-create-one-interface-endpoint)?

Reply:
Hi @themantissa , the url it's different, the docs it based from this https://docs.atlas.mongodb.com/reference/api/private-endpoints-endpoint-create-one which it's the same as interface endpoint deprecated

Reply:
I know, I put that URL first and if you look at the status doc for AWS it is different between the old and new.